custom_components/yamaha_ynca/__init__.py

```
@@ -92,12 +96,10 @@ def build_zone_devicename(receiver, zone_subunit):
         devicename = zone_subunit.zonename
     return devicename
 
+
 def build_zoneb_devicename(receiver):
```

Ruff (0.8.2)

Missing return type annotation for public function `build_zoneb_devicename`

(ANN201)